File: C/Lista_01(Estrutura_Sequencial)/Circulo.c
#include <stdio.h>
#include <math.h>

int main () {

    double raio;

    printf("Digite o valor do raio do circulo: ");
    scanf("%lf", &raio);

    double area = pow(raio,2)*3.14159;

    printf("AREA = %.3lf", area);

}

File: Java/Lista_01(Estrutura_Sequencial)/Circulo.java
import java.util.Locale;
import java.util.Scanner;

public class Circulo {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner s = new Scanner(System.in);
		
		System.out.print("Digite o valor do raio do circulo: ");
		double raio = s.nextDouble();
		
		double area = Math.pow(raio, 2) * Math.PI;
		
		System.out.printf("AREA = %.3f", area);
		
		s.close();
		
	}

}
